azure-quantum/azure/quantum/optimization/problem.py

```diff
@@ -124,12 +127,46 @@ def add_term(self, c: Union[int, float], indices: List[int]):
         self.terms.append(Term(indices=indices, c=c))
         self.uploaded_blob_uri = None
 
-    def add_terms(self, terms: List[Term]):
-        """Adds a list of terms to the `Problem` representation
+    def add_terms(self, terms: List[Term],
+                  type: str = None, c: Union[int, float] = 1):
+        """Adds an optionally grouped list of monomial terms 
+        to the `Problem` representation
 
         :param terms: The list of terms to add to the problem
+        :param type: Type of grouped term being added, if applicable
+        :param c: Weight of grouped term, if applicable
+        """
+        if type:
+            # Grouped term
+            if type == 'na':
+                gtype = GroupType.combination
```

Comment:
what's the difference between this and no type?

Reply:
No type means that the list of terms are added without being grouped into a face, which means that (if there are no other grouped terms), the model used for that cost function will be the more basic ones (without faces or SLC terms, etc). Having type 'na' is one way to trigger the SLC models even with a cost function that doesn't need it (and is built into the SLC models anyway, so I think it's good practice to have this accessible via QDK).